docs/semgrep-secrets/historical-scanning.md

```diff
@@ -0,0 +1,119 @@
+---
+slug: historical-scanning
+append_help_link: true
+title: Scan your Git history (beta)
+hide_title: true
+description: Detect valid, leaked secrets in previous Git commits through a historical scan.
+tags:
+  - Semgrep Secrets
+  - Team & Enterprise tier
+---
+
+# Scan your Git history (beta)
+
+Detect valid, leaked secrets in previous Git commits through a **historical scan**.
+
+You can perform one-time historical scans or enable historical scanning for full Secrets scans. Detecting valid secrets in your Git history is a step to reducing your repository's attack surface.
+
+You can run historical scans in the CLI and in your Semgrep deployment, which enables you to track and triage these secrets.
```

Comment:
what does "and in your Semgrep deployment" mean?

Reply:
We use the term "deployment" to refer to the totality of your Semgrep setup, because we'll be supporting ZCS, which doesn't use a CI provider such as GitHub Actions.